.travis.yml

```diff
@@ -30,7 +30,7 @@ matrix:
       env: TASK=code-lint
       # Running Code Linter -- Requires @loopback/build so it's bootstrapped
       script:
-        - lerna bootstrap --scope @loopback/build --include-filtered-dependencies
+        - lerna bootstrap --scope @loopback/build --include-dependencies
```

Comment:
<img width="752" alt="Screen Shot 2019-10-18 at 12 21 21 AM" src="https://user-images.githubusercontent.com/42985749/67065804-b399a200-f13d-11e9-9330-bb78aa84ae09.png">

Replacing because `--include-filtered-dependencies` has been renamed.